tests/integration/fuzzing_strategies/short_strings_test.cairo

```diff
@@ -0,0 +1,13 @@
+%lang starknet
+
+@external
+func setup_correct_short_string() {
+    %{ given(value=strategy.short_strings()) %}
+    return ();
+}
+
+@external
+func test_correct_short_string{syscall_ptr: felt*, range_check_ptr}(value: felt) {
+    %{ assert 0 <= ids.value %}
+    return ();
+}
```

Comment:
This test doesn't check if the value is less than the short string max possible value.

- [ ] add assert checking the short string value is in the expected range

Reply:
Done in [276916c](https://github.com/software-mansion/protostar/pull/886/commits/276916c3f08b4704d6f5cc30a21b1e632d221abe)